### DISCUSSION TITLE
Pubs/pages take a long time to load

### DISCUSSION THREAD

**Discussion by kaciccone** [knowledgefutures/pubpub#2179] [Support]
Often when I click on a link from a collection page to a pub, or vice versa, it usually takes approximately three seconds to load. That is a long time. However, sometimes it takes even longer. Is there a reason why pages sometimes take a long time to load?

**Comment by gabestein** [comment]
Hi Karen, sorry you're having this issue. Can you share the urls of the pages/pubs where you're seeing this issue? And to clarify, does it happen when you're visiting the dashboards for them, or the public-facing releases/pages?

**Reply by kaciccone** [reply]
This has only happened a couple of times since I submitted this issue. Once I was in pub settings and clicked to go to the pub, and it took about 7 seconds. Most of the time it takes about 3 seconds to go from a dashboard to a pub, or from a collection page to a pub. I'll document details and let you know if the problem recurs.
